wsmaster/che-core-api-devfile/pom.xml

```diff
@@ -155,7 +155,7 @@
                 <version>1.4</version>
                 <executions>
                     <execution>
-                        <id>test</id>
+                        <id>compile</id>
```

Comment:
Something more descriptive like `add-json-schema-generated-sources`?

Reply:
Thanks for pointing this.
Well. I faced an issue that IDE failed to resolve Devfile classes after `mvn clean install -DskipTests` and I thought it references maven phase and it does not generate sources when tests are disabled.
Now I see this id is just identifier of execution and it does not reference maven phase. So, looks like this PR fixes nothing )

Reply:
It even works after removing this execution at all